challenges/unit-testing/passing-tests/get-average/get-average.js

```diff
@@ -2,6 +2,13 @@
 // return the average of all the numbers
 // be sure to exclude the strings
 
-function average(numbers) {}
+function average(numbers) {
+  const onlyNumbers = numbers.filter((item) => typeof item === "number");
```

Comment:
What happens on ["hello"]?

Reply:
```hello``` is ignored because it is not a number.

Reply:
Avoid dividing by zero. In some other programming languages it would lead to a crash